### PR TITLE
Add metadata url for Community Care Health API

### PR DESCRIPTION
**Pull request by michaelkimball**
### Description

For [API-9818](https://vajira.max.gov/browse/API-9818) switching all APIs to use the metadata URL and put a fake version in Staging.

### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [ ] Accessibility concerns have been considered and addressed
  
  - [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

File: .env.staging [hunk 1]
```diff
@@ -12,4 +12,4 @@ REACT_APP_SHOW_TESTING_NOTICE=true
 REACT_APP_CLAIMS_ATTRIBUTES_API_ENABLED=false
 REACT_APP_CONSUMER_DOCS=true
 REACT_APP_FHIR_API_ENABLED=false
-REACT_APP_COMMUNITY_CARE_API_ENABLED=false
+REACT_APP_COMMUNITY_CARE_API_ENABLED=true
```
Comment (christineyost) [inline .env.staging:15]: Could also completely remove this environment variable, but not sure if that's more or less confusing

**Review by christineyost — APPROVED** [review]
Looks good to me!

File: .env.staging (hunk 1 above)
Comment (michaelkimball) [reply .env.staging:15]: Good call

**Review by christineyost — APPROVED** [review]
🚀

**Review by mheusgit — APPROVED** [review]
Looks fine

**Pull request merged**